patternlab/styleguide/source/_patterns/03-organisms/by-author/account-manager.twig

```diff
@@ -0,0 +1,19 @@
+<section class="ma__account-manager">
+  {# Static title, users can not change #}
+  <h3 id="manage-account" class="ma__comp-heading">Manage your account</h3>
+
+  
+  <div class="ma__account-manager--sections">
+    {% for section in accountManagementSections %}
+    <div class="ma__account-manager--section">
```

Comment:
Can this be moved to a separate pattern (lines 8 - 14) and wrapped in a block? This would give us the ability to embed an account manager on a page and over ride the inside of the loop so that we can map individual instance directly.  Also, I believe it makes it more readable to add an indention between a for loop and the lines inside.